src/router/Router.jsx

```diff
@@ -1,16 +1,14 @@
 import React from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
-import NavBar from "../components/NavBar";
```

Comment:
🔧 Can you remove this component's files from the project, since we aren't using it anymore?

Reply:
files removed